components/shared/DayRangePicker.tsx

```diff
@@ -40,9 +40,10 @@ export const DayRangePicker = ({ onDayRangeChanged }: DayRangePickerProps) => {
   };
 
   return (
-    <label className="w-fit font-semibold">
+    <label htmlFor="day-range-select" className="w-fit font-semibold">
```

Comment:
These labels wrap the button which is the `<SingleSelect />` trigger. Buttons don't need labels, but they do need button text, so you can add a `labelText: string` prop that is required and then you can do this.

![CleanShot 2024-10-21 at 10 37 02](https://github.com/user-attachments/assets/7ea0d374-1791-4af1-8556-81c911665c5e)

Then you can replace the labels with sr-only text that wrap the `<SingleSelect />` instances with a `labelText` prop on the `<SingleSelect />` instances.